R/conda.R

```diff
@@ -194,7 +194,8 @@ conda_version <- function(conda = "auto") {
 }
 
 
-
+#' @rdname conda-tools
+#' @export
 find_conda <- function() {
```

Comment:
This function finds the global conda rather than the currently in use conda environment so I don't think it's what you want.